src/psycopmlutils/loaders/raw/load_diagnoses.py

```diff
@@ -254,3 +254,370 @@ def sleep_problems_unspecified(n: Optional[int] = None) -> pd.DataFrame:
+            icd_code=["f30", "f31"],
+            wildcard_icd_code=True,
+            n=n,
+        )
+
+    @data_loaders.register("depressive disorders")
+    def depressive_disorders(n: Optional[int] = None) -> pd.DataFrame:
+        return LoadDiagnoses.from_physical_visits(
+            icd_code=["f32", "f33", "f34", "f38"],
+            wildcard_icd_code=True,
+            n=n,
+        )
+
+    @data_loaders.register("other affective disorders")
+    def other_affective_disorders(n: Optional[int] = None) -> pd.DataFrame:
+        return LoadDiagnoses.from_physical_visits(
+            icd_code=["f38", "f39"],
+            wildcard_icd_code=True,
+            n=n,
+        )
+
+    # data loaders for all diagnoses in the f4-chapter (neurotic, stress-related and somatoform disorders).
+
+    @data_loaders.register("all f4-disorders")
+    def all_f4_disorders(n: Optional[int] = None) -> pd.DataFrame:
+        return LoadDiagnoses.from_physical_visits(
+            icd_code="f4",
+            wildcard_icd_code=True,
+            n=n,
+        )
+
+    @data_loaders.register("phobic,anxiety and ocd")
+    def phobic_and_anxiety(n: Optional[int] = None) -> pd.DataFrame:
+        return LoadDiagnoses.from_physical_visits(
+            icd_code=["f40", "f41", "f42"],
+            wildcard_icd_code=True,
+            n=n,
+        )
+
+    @data_loaders.register("reaction to severe stress and adjustment disorders")
+    def stress_and_adjustment(n: Optional[int] = None) -> pd.DataFrame:
+        return LoadDiagnoses.from_physical_visits(
+            icd_code="f43",
+            wildcard_icd_code=True,
+            n=n,
+        )
+
+    @data_loaders.register("dissociative, somatoform and others")
+    def dissociative_somatoform_and_others(n: Optional[int] = None) -> pd.DataFrame:
+        return LoadDiagnoses.from_physical_visits(
+            icd_code=["f44", "f45", "f48"],
+            wildcard_icd_code=True,
+            n=n,
+        )
+
+    # data loaders for all diagnoses in the f5-chapter (behavioural syndromes associated with physiological disturbances and physical factors).
+
+    @data_loaders.register("all f5-disorders")
+    def all_f5_disorders(n: Optional[int] = None) -> pd.DataFrame:
+        return LoadDiagnoses.from_physical_visits(
+            icd_code="f5",
+            wildcard_icd_code=True,
+            n=n,
+        )
+
+    @data_loaders.register("eating disorders")
+    def eating_disorders(n: Optional[int] = None) -> pd.DataFrame:
+        return LoadDiagnoses.from_physical_visits(
+            icd_code="f50",
+            wildcard_icd_code=True,
+            n=n,
+        )
+
+    @data_loaders.register("sleeping and sexual disorders")
+    def sleeping_and_sexual_disorders(n: Optional[int] = None) -> pd.DataFrame:
+        return LoadDiagnoses.from_physical_visits(
+            icd_code=["f51", "f52"],
+            wildcard_icd_code=True,
+            n=n,
+        )
+
+    @data_loaders.register("f5 others")
```

Comment:
These function names are perhaps also a bit problematic, because the user would have no idea what `f5_others` include. I, however, don't have a great solution. @MartinBernstorff, maybe you have an idea?

Reply:
I agree, and I think you'll also have problems in reporting. Whatever unifies these remaining F5's is what I'd call the function.

Maybe "misc. F5", since the diagnoses are really a misc. bin.